internal/rbd/nodeserver.go

```diff
@@ -1314,6 +1317,55 @@ func (ns *NodeServer) xfsSupportsReflink() bool {
 	return false
 }
 
+// ext4SupportsPrezeroed checks if the ext4 filesystem supports the
+// "assume_storage_prezeroed" option. It does this by creating a temporary
+// image file, attempting to format it with the ext4 filesystem using the
+// "assume_storage_prezeroed" option, and checking the output for errors.
+func (ns *NodeServer) ext4SupportsPrezeroed() bool {
```

Comment:
how about if we check if this is supported or not in the driver.go(if we are stating it as nodeplugin) before starting the node plugin and store it in the Nodeserver struct like its suggested here https://github.com/ceph/ceph-csi/pull/4996/files#r1910308642?

Reply:
As the result for this is cached, let's check it when we need it instead of doing it preemptively? This way also has the added benefit of keeping `driver.go` simple. WDYT?